test/tools/router_check/validation.proto

```diff
@@ -65,6 +65,9 @@ message ValidationInput {
   // The “:authority”, “:path”, “:method”, “x-forwarded-proto”, and “x-envoy-internal” fields are
   // specified by the other config options and should not be set here.
   repeated envoy.api.v2.core.HeaderValue additional_headers = 8;
+
+  // Enable a runtime flag
```

Comment:
Don't you want to allow a percentage here so that the user can test different decisions? I'm a little confused about the semantics. Can you clarify in the documentation more?

Reply:
@mattklein123 The percentage is guided by the existing `random_value` property.

Reply:
Makes sense, thanks. Can you add more docs here to indicate that the random_value property is used to effect the settings?